packages/db/schema/sqlite.ts

```diff
@@ -297,6 +297,11 @@ export const iconRepositories = sqliteTable("iconRepository", {
   slug: text("iconRepository_slug").notNull(),
 });
 
+export const serverSettings = sqliteTable("serverSetting", {
+  settingKey: text("serverSetting_settingKey").notNull().unique().primaryKey(),
```

Comment:
The name of the column seems redundant (`settingKey` could be `key` and `serverSetting_settingKey` could be `key`)

Reply:
So ``serverSetting_key`` or ``key``? As far as I can see, we use the prefix in all places

Reply:
just `key`. See integrations for example: 
![image](https://github.com/homarr-labs/homarr/assets/63781622/047053ff-d820-42cb-9e7a-204ec2267da1)